Purchases/Logging/Strings/PurchaseStrings.swift

```diff
@@ -49,6 +49,10 @@ enum PurchaseStrings {
     case product_unpurchased_or_missing
     case transaction_unverified(productID: String, errorMessage: String)
     case begin_refund_request_unsupported
+    case begin_refund_for_entitlement_nil_customer_info(entitlementID: String?)
```

Comment:
i eventually would like to make these sub-errors

Reply:
story here: https://app.shortcut.com/revenuecat/story/10784/ios-refactor-beginrefundrequest-to-use-suberrors-instead-of-strings-for-specific-errors